### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by septhyme**
This file has been edited by a Computing Science student in Technical Writing course. The students are tasked to revise a documentation or technical information on GitHub. Students will get full marks if the author accepts the changes in the pull request.

Please review my documentation and accept the request if necessary. I am ready to edit or add more to the documentation if you feel the revision is not well written.

Thank you for your time.

### CODE REVIEW

**Comment by kordianbruck** [comment]
Thanks for the suggestions!

@septhyme its a lot more technical in our wiki if you are up for a challenge: https://github.com/TCA-Team/TumCampusApp/wiki

**Comment by septhyme** [comment]
Hi @kordianbruck,

I am sorry that I am unable to put more details on the wiki page because I am not sure about how the program works. But, I can check the wiki page if you need it.

Thank you for your attention!

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # TumCampusApp [![Build Status](https://server4.wdkk.de/buildStatus/icon?job=TCA-Android)](https://server4.wdkk.de/job/TCA-Android/) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/01c41ee04d684a11947bfd5216de0b1c)](https://www.codacy.com/app/TCA-Bot/TumCampusApp?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=TCA-Team/TumCampusApp&amp;utm_campaign=Badge_Grade)
 ### An excellent app for students by students
-The TUM Campus App (TCA) is a open source project, mainly developed by volunteers with significant contributions by the Android Practical course. The TCA mostly targets phones, but can also be used on tablets or any other device that runs Android.
+The TUM Campus App (TCA) is an open source project, which is mainly developed by volunteers with significant contributions by the Android Practical course. The TCA mostly targets phones, but can also be used on tablets or any android device.
```
Comment (pfent) [inline README.md]: Android is a name, so should be written uppercase

**Comment by pfent** [comment]
While you're here, would you mind looking over our in-App strings in [English](https://github.com/TCA-Team/TumCampusApp/blob/master/app/src/main/res/values/strings.xml) or [German](https://github.com/TCA-Team/TumCampusApp/blob/master/app/src/main/res/values-de/strings.xml)?

Apart from that, thanks a lot for the corrections :heart:, our documentation is certainly lagging behind a bit

**Comment by septhyme** [comment]
Hello @pfent,

Yes, I would be glad to take a look at the in-app string in English, but I cannot do the German version since I am not proficient in the German language. I also changed the readme file. 

Thank you for your reply!

**Pull request merged**